test/unit/org/apache/cassandra/cql3/validation/operations/CreateTest.java

```diff
@@ -365,8 +367,9 @@ public void testKeyspace() throws Throwable
 
         execute("CREATE KEYSPACE testXYZ WITH replication = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }");
 
+        String longKeyspace = IntStream.range(1, 230).mapToObj(i ->"x").collect(Collectors.joining());
```

Comment:
lets use `SchemaConstants.NAME_LENGTH + something` instead of 230 *or* add an assert that `230 > SchemaConstants.NAME_LENGTH`

and lets rename `longKeyspace` to `tooLongKeyspace` ;)